pl/python_introduction/README.md

```diff
@@ -12,13 +12,18 @@ Aby rozpocząć zabawę z Pythonem, musisz otworzyć jego *wiersz poleceń* na s
 
 Gdy już będziesz gotowa, postępuj według instrukcji poniżej.
 
-Chcemy otworzyć konsolę Pythona. Wpisz `python`, jeśli pracujesz na Windowsie, lub `python3`, jeśli pracujesz na MacOS/Linuxie. Wciśnij `Enter`.
+Chcemy otworzyć konsolę Pythona. 
+
+Wpisz `python`, jeśli pracujesz na Windowsie, lub `python3`, jeśli pracujesz na MacOS/Linuxie. Wciśnij `Enter`.
 
     $ python3
     Python 3.4.3 (...)
```

Comment:
Hi @magul 
1. i could be more engaged in july, so then i could update also en verision,
2. Python version - windows installer looks different in version 3.5 and 3.6 than it used to be like in 3.4, so i had few confused girls when i was mentoring - so i thought it could be updated with more details.

In conclusion, we can update en version first, and than pl, but as i wrote it earlier till end of june i won't have time :(